components/Ink/src/MDCInkView.h

```diff
@@ -54,27 +54,36 @@ typedef NS_ENUM(NSInteger, MDCInkStyle) {
 @property(nonatomic, assign) MDCInkStyle inkStyle;
 
 /** The foreground color of the ink. The default value is defaultInkColor. */
-@property(nonatomic, strong, null_resettable) UIColor *inkColor;
+@property(nonatomic, strong, nonnull) UIColor *inkColor UI_APPEARANCE_SELECTOR;
 
 /** Default color used for ink if no color is specified. */
 @property(nonatomic, strong, readonly, nonnull) UIColor *defaultInkColor;
 
 /**
  Maximum radius of the ink. If the radius <= 0 then half the length of the diagonal of self.bounds
  is used. This value is ignored if @c inkStyle is set to |MDCInkStyleBounded|.
+
+ Ignored if updated ink is used.
  */
 @property(nonatomic, assign) CGFloat maxRippleRadius;
 
+/**
+ Use the older legacy version of the ink ripple. Default is YES.
+ */
+@property(nonatomic, assign) BOOL usesLegacyInkRipple;
+
 /**
  Use a custom center for the ink splash. If YES, then customInkCenter is used, otherwise the
  center of self.bounds is used. Default is NO.
+
+ Only used with legacy ink. Ignored if updated ink is used.
```

Comment:
Text copy suggestion:

> Affects behavior only if `usesLegacyInkRipple` is enabled.

Intent is to avoid introducing a new concept of "updated ink" here.

Reply:
Done.